src/Voron/Util/DataCopier.cs

```diff
@@ -36,23 +36,19 @@ public void ToStream(byte* ptr, long count, Stream output)
             }
         }
 
-        public void ToStream(AbstractPager src, long startPage, long numberOfPages, 
-            Stream output, Action<string> infoNotify, CancellationToken cancellationToken)
+        public static void ToStream(AbstractPager src, long startPage, long numberOfPages, Stream output, Action<string> infoNotify, CancellationToken cancellationToken)
         {
             // In case of encryption, we don't want to decrypt the data for backup, 
             // so let's work directly with the underlying encrypted data (Inner pager).
-         
-            if ((_buffer.Length % Constants.Storage.PageSize) != 0)
-                throw new ArgumentException("The buffer length must be a multiple of the page size");
 
-            var steps = _buffer.Length/ Constants.Storage.PageSize;
+            var toCopyInBytes = new Size(1, SizeUnit.Megabytes).GetValue(SizeUnit.Bytes);
```

Comment:
`public DataCopier(int bufferSize)` -> shouldn't we set the buffer size there and use this value here?